core/dbt/tree_sitter_jinja/__init__.py

```diff
@@ -0,0 +1 @@
+from . import extractor  # noqa
```

Comment:
(optional) I want to say you can remove this line, I believe it is redundant if I understand how python importing work.

Reply:
You're right! I love the chance to remove more code.